tta-server/app/models/Population.scala

```diff
@@ -0,0 +1,24 @@
+package models
+
+object Population {
+
+  def generateIncreasePopulationAction(playerState: PlayerState): Action = new Action{
```

Comment:
Oh, but you're right, doIt already has access to gameState so the Action itself doesn't need it.